cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectClusterTest.java

```diff
@@ -1469,12 +1461,6 @@ public void testOpenTelemetryTracing() {
     public void testTracing(String type, String consumerInterceptor, String producerInterceptor) {
         KafkaConnectBuilder builder = new KafkaConnectBuilder(this.resource);
         switch (type) {
-            case JaegerTracing.TYPE_JAEGER:
```

Comment:
Same as above - isn't this method used only from one place? MAybe we can get rid of it? It seems to be also in the other tests.